plans/chew-large-datasets/main.go

```diff
@@ -21,6 +26,33 @@ func main() {
 		panic("test case sequence number not set")
 	}
 
-	// Demux to the right test case.
-	testCases[runenv.TestCaseSeq](runenv)
+	name := runenv.TestCase
+	tc := testCases[runenv.TestCaseSeq]
+
+	cfg, err := utils.GetTestConfig(runenv)
+	if err != nil {
+		runenv.Abort(fmt.Errorf("could not retrieve test config: %s", err))
+		return
+	}
+
+	// Test with the API
+	runenv.TestCase = name + "-api"
```

Comment:
```suggestion
	runenv.TestCase = name + "-core-api"
```

-api is rather generic, -core-api is not better but at least we know what is its referring to at all times